docs/contribute/cumulative-docs/example-scenarios.md

```diff
@@ -0,0 +1,545 @@
+page from the Elastic Stack in 9.1.0 and from Serverless around the same time.
+Since this this functionality is still available before 9.1.0, we need that content to continue to be
+available to users on Elastic Stack earlier versions while communicating to users on newer versions
+that it is no longer available.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-removed-unversioned-exception.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+applies_to:
+  stack: all
+  serverless:
+    security: all
+---
+
+# Configure advanced settings [security-advanced-settings]
+
+[...]
+
+## Access the event analyzer and Session View from the event or alert details flyout [visualizations-in-flyout]
+
+```{applies_to}
+stack: removed 9.1
+serverless: removed
+```
+
+[...]
+````
+::::
+:::::
+
+
+::::::{important}
+If the functionality was only ever available in an unversioned product or deployment mode,
+remove the content altogether.
+
+% FOR THE REVIEWER: IS THERE AN EXAMPLE?
+% I couldn't find an actual example of this scenario.
+% For example, we removed the <something> functionality that was described on the [<page>](#) page
+% from the Elastic Stack in 9.1.0.
+% Since this this functionality is still available in 9.0.0, we need that content to continue to be
+% available to users on Elastic Stack 9.0.0 while communicating to users on newer versions that it is no longer available.
+% :::::{tab-set}
+% ::::{tab-item} Image
+% :::{image} ./images/example-not-one-section.png
+% :screenshot:
+% :alt:
+% :::
+% ::::
+% ::::{tab-item} Code
+% ::::
+% :::::
+::::::
+
+## Code block content varies [code-block]
+
+Often the content in a code block will vary between situations (versions, deployment types, etc).
+There are a couple possible solutions.
+
+### Solution A: Use a code callout [code-block-callout]
+
+Using a code callout is the lightest-touch solution, but might not be sufficient in all cases.
+
+**When to use a code callout**:
+
+* The code block and its callouts fit vertically on a typical laptop screen.
+  This will reduce the risk of users copying the code snippet without reading the information in the callout.
+* Syntax is either just added or just removed — syntax is not modified.
+  It is difficult to communicate that some syntax is needed in more than one situation but varies depending on the situation.
+* The code block will not require more than 3 `applies_to`-related callouts.
+  At that point, the code becomes more difficult to read and use.
+
+**Best practices**:
+
+* Place the badge at the beginning of the callout.
+
+**Example**: On the [Entity Analytics](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-entity-analytics#_configuration_3) page, we added a new option to a code block that was only made available in 9.1.0.
+
+::::{image} ./images/example-code-block-callout.png
+:screenshot:
+::::
```

Comment:
the ideal scenario is `[feature] was introduced in [version]`

but our inline badges look terrible in sentences ... 

but the takeway is that I don't think the current recommended prose is clear